### PR TITLE
Add Vale checks to SSG

### PR DESCRIPTION
**Pull request by bburt-rh**
This PR adds Vale checks to the SSG repo

### CODE REVIEW

**Comment by aireilly** [comment]
add this in `supplementary-style-guide/supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini` to tweak how Vale runs in the Glossary files (YES/NO on rules)

```
StylesPath = ../../../.vale/styles

# Ignore files in dirs starting with `.` to avoid raising errors for `.vale/fixtures/*/testinvalid.adoc` files
[[!.]*.adoc]
BasedOnStyles = RedHat, AsciiDoc

MinAlertLevel = suggestion

RedHat.Abbreviations = NO
RedHat.CaseSensitiveTerms = NO
RedHat.Conjunctions = NO
RedHat.ConsciousLanguage = NO
RedHat.Contractions = YES
RedHat.Definitions = NO
RedHat.Ellipses = NO
RedHat.HeadingPunctuation = NO
RedHat.Headings = YES
RedHat.OxfordComma = YES
RedHat.PascalCamelCase = YES
RedHat.PassiveVoice = NO 
RedHat.ReadabilityGrade = NO
RedHat.ReleaseNotes = NO
RedHat.RepeatedWords = YES
RedHat.SelfReferentialText  = NO
RedHat.SentenceLength = NO  
RedHat.SimpleWords = NO
RedHat.Slash = YES
RedHat.Spacing = YES
RedHat.Spelling = NO
RedHat.TermsErrors = NO
RedHat.TermsSuggestions = YES
RedHat.TermsWarnings = NO
RedHat.Usage = NO
RedHat.UserReplacedValues = YES
```

**Comment by aireilly** [comment]
Remove this `supplementary-style-guide/supplementary_style_guide/.vale/styles/` folder

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (aireilly) [inline .vscode/settings.json:1]: This file shouldn't be here.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+RedHat.Conjunctions = NO
+RedHat.ConsciousLanguage = NO
+RedHat.Contractions = YES
+RedHat.Definitions = NO
+RedHat.Ellipses = NO
+RedHat.HeadingPunctuation = NO
+RedHat.Headings = YES
+RedHat.OxfordComma = YES
+RedHat.PascalCamelCase = YES
+RedHat.PassiveVoice = NO
```
Comment (aireilly) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini:19]:
```suggestion
RedHat.PassiveVoice = YES
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini [hunk 3]
```diff
@@ -0,0 +1,33 @@
+StylesPath = ../../../.vale/styles
+
+[[!.]*.adoc]
+BasedOnStyles = RedHat, AsciiDoc
+
+MinAlertLevel = suggestion
+
+RedHat.Abbreviations = NO
+RedHat.CaseSensitiveTerms = NO
+RedHat.Conjunctions = NO
```
Comment (aireilly) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini:10]:
```suggestion
RedHat.Conjunctions = YES
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+RedHat.HeadingPunctuation = NO
+RedHat.Headings = YES
+RedHat.OxfordComma = YES
+RedHat.PascalCamelCase = YES
+RedHat.PassiveVoice = NO
+RedHat.ReadabilityGrade = NO
+RedHat.ReleaseNotes = NO
+RedHat.RepeatedWords = YES
+RedHat.SelfReferentialText  = NO
+RedHat.SentenceLength = NO
```
Comment (aireilly) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini:24]:
```suggestion
RedHat.SentenceLength = YES
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+MinAlertLevel = suggestion
+
+RedHat.Abbreviations = NO
+RedHat.CaseSensitiveTerms = NO
+RedHat.Conjunctions = NO
+RedHat.ConsciousLanguage = NO
+RedHat.Contractions = YES
+RedHat.Definitions = NO
+RedHat.Ellipses = NO
+RedHat.HeadingPunctuation = NO
```
Comment (aireilly) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/.vale.ini:15]:
```suggestion
RedHat.HeadingPunctuation = YES
```

**Comment by mportman12** [comment]
@bburt-rh What's the status of this PR? Do we know what the blocker is, and is there a plan on how to resolve it?

**Comment by aireilly** [comment]
@bburt-rh I can help with this, reach out if you are stuck. You've got the main bits and pieces in place, I can see one or two other things to tweak.

**Comment by aireilly** [comment]
I've added a PR to Brian's fork which updates this work: https://github.com/bburt-rh/supplementary-style-guide/pull/1

Once this is accepted/merged, IMO this is good to go. Details of the errors this update allows for the SSG are here: http://pastebin.test.redhat.com/1107625 (needs VPN)

**Pull request closed without merging**